custom/icds_reports/utils/aggregation_helpers/awc_location.py

```diff
@@ -100,6 +100,22 @@ def aww_query(self):
             ucr_aww_tablename=self.ucr_aww_tablename
         )
 
+    def ls_query(self):
+        return """
+            UPDATE "{tablename}" awc_loc SET
+              ls_name = ut.username
+            FROM (
+              SELECT
+                commcare_location_id,
+                username
+              FROM "{ucr_aww_tablename}"
+            ) ut
+            WHERE ut.commcare_location_id = awc_loc.supervisor_id
```

Comment:
This could be simplified to 
```
            UPDATE "{tablename}" awc_loc SET
              ls_name = ut.username
            FROM "{ucr_aww_tablename}" ut
            WHERE ut.commcare_location_id = awc_loc.supervisor_id
```
No?

Reply:
yeah, there are number of other columns also, did not want to pull out entire data,  just fetched the required column.